contracts/payloads/IGP89/PayloadIGP89.sol

```diff
@@ -368,7 +375,29 @@ contract PayloadIGP89 is PayloadIGPMain {
         }
 
     }
+    // @notice Action 9: Update lower range of wstETH-ETH DEX
+    function action6() internal {
+        address wstETH_ETH_DEX_ADDRESS = getDexAddress(1);
+
+        {
+            // Update Lower Range
+            IFluidDex(wstETH_ETH_DEX_ADDRESS).updateRangePercents(
+                0.09 * 1e4, // 0.09%
+                0.1 * 1e4, // 0.1%
```

Comment:
upper should stay at `1` 

what Samyak meant was change to 0.1 lower in 2 steps as the shifting must be instant